### ISSUE TITLE
Restore home breadcrumb to Outreach Hub event pages

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-team#16020]
## What happened?
Breadcrumb errors across all event pages. This FE template is used on two products (VAMC system and Outreach Hub), which have slightly different requirements. Correlated VAMC breadcrumb issue tracked in #9439

1. The breadcrumb shows the spoke label (instead of Home link) 
2. The middle breadcrumb is sometimes displayed as "Outreach events" and sometimes displayed as only "Events". 

**Hyphenation issues will be addressed in #15880** 

## Steps to Reproduce
(Revised to use new event since original events reported have expired /return 404) 

**Outreach event -  https://www.va.gov/outreach-and-events/events/46993/**
Breadcrumbs: `Home > Outreach and events > Events > Women Veteran's Day Celebration`
Should be: `Home > Outreach and events > Outreach events > Women Veteran's Day Celebration`

<details>

![image](https://user-images.githubusercontent.com/85581471/172949940-6d8da297-370b-49f0-a6a4-c486d55d76b1.png)

Since the time this ticket was created (or due to a bug?) Outreach events no longer show a left sidebar.
Clicking the "Events" crumb points to https://www.va.gov/outreach-and-events/events/, where the crumb is correctly showing as:
`Home > Outreach and Events > Outreach events`

![image](https://user-images.githubusercontent.com/85581471/172950481-58498a12-e6e5-4317-b569-a8bc6652ae2d.png)

</details>

## Acceptance Criteria
- [x] **Hyphenation issues will be addressed in #15880** 
- [x] The Home breadcrumb is restored on non-VAMC event pages
- [ ] Ensure the breadcrumb = Home > Outreach and events > **Outreach events**

**Comment by brangi** [comment]
@mmiddaugh 

> 2. The middle breadcrumb is sometimes displayed as "Outreach events" and sometimes displayed as only "Events".

This is a data issue:


<img width="1425" alt="Screen Shot 2021-02-01 at 6 06 15 PM" src="https://user-images.githubusercontent.com/100468/106533809-68350f80-64b8-11eb-859a-f8f63d5caeba.png">

**Comment by brangi** [comment]
@mmiddaugh for this issue:

> 
> 1. The breadcrumb shows the spoke label (instead of Home link)

We need another value from the cms data that tells the frontend when to hide the breadcrumbs or at least know that the page is VAMC. Otherwise we can only hardcode the page we need to hide the home breadcrumb and show it. It is not scalable. This is because we use the same template for different pages.

**Comment by mmiddaugh** [comment]
This issue is blocked by Breadcrumb cross-team spike (with Mikki, Public website engineers, CMS, Facilities engineers)

**Comment by jilladams** [comment]
Modern test: 

**Outreach event -  https://www.va.gov/outreach-and-events/events/46993/**
Breadcrumbs: `Home > Outreach and events > Events > Women Veteran's Day Celebration`
* Home => https://www.va.gov/
<details>

![image](https://user-images.githubusercontent.com/85581471/172949940-6d8da297-370b-49f0-a6a4-c486d55d76b1.png)

Since the time this ticket was created (or due to a bug?) Outreach events no longer show a left sidebar.
Clicking the "Events" crumb points to https://www.va.gov/outreach-and-events/events/, where the crumb is correctly showing as:
`Home > Outreach and Events > Outreach events`

![image](https://user-images.githubusercontent.com/85581471/172950481-58498a12-e6e5-4317-b569-a8bc6652ae2d.png)

</details>


**VAMC event - https://www.va.gov/jackson-health-care/events/46958/**
Breadcrumbs: `Home > Jackson health care > Events > 3rd Annual Virtual Caregiver Support Summit`
* Home => https://www.va.gov/ -- should instead point to the System page, per this ticket description.

<details>
![image](https://user-images.githubusercontent.com/85581471/172949964-5985e1bb-c7ac-4f63-bf99-a771c0bd197d.png)

</details>

**Comment by jilladams** [comment]
Per Dave: currently testing a breadcrumb hypothesis on Vet Centers to replace Home with VA.gov home, e.g. https://www.va.gov/boston-vet-center/. Will become a sitewide change if it proves out
![Screen Shot 2022-06-10 at 12 18 12 PM](https://user-images.githubusercontent.com/85581471/173135294-01eb9fec-bc55-4201-876f-6b1987ef883f.png)

**Comment by jilladams** [comment]
Per 6/10 Events backlog scrub, this ticket can be prioritized now for Outreach events. 

VAMC events broken into separate ticket, #9439 , and removed from scope of this ticket.

**Comment by jilladams** [comment]
closing as dupe of https://github.com/department-of-veterans-affairs/va.gov-cms/issues/9439
